checkers-flex.html

```diff
@@ -6,11 +6,14 @@
   <link rel="stylesheet" href="checkers-flex-style.css" type="text/css">
 </head>
 <body>
+  <div id="game">
+  <h4>Player 1: {{ player1.name }}, chip color: {{ player1.color }}</h4>
+  <h4>Player 2: {{ player2.name }}, chip color: {{ player2.color }}</h4>
```

Comment:
here I am adding some player stats, and making sure that my Vue instance is reading the data by outputting the variables in *mustache syntax*: `{{ player1.name }}`